### PR TITLE
[Feature]: creates auth pages mockup

### PR DESCRIPTION
**Pull request by obayomi96**
#### What does this PR do?
- create signin page
- create parent signup page
- create student signup page

#### How should this be manually tested?
- To view signin page visit `/signin`
- To view parent register page visit `/parent-signup`
- To view student register page visit `/student-signup`

#### Any background context OR Extra detail?
- I did not add placeholdersin the input fields on the signup pages because we cant have both Label and placeholders.

#### What are the relevant PT stories?
- Signin [Link](https://www.pivotaltracker.com/story/show/171986606)
- Signup stories [Link](https://www.pivotaltracker.com/story/show/171986614)

#### Screenshot
Screenshot from 2020-04-08 16-59-02
Screenshot from 2020-04-08 16-58-57
Screenshot from 2020-04-08 16-58-47

### CODE REVIEW

File: src/views/Login/index.js [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+          </div>
+          <input className="submit" type="submit" value="Sign in" />
+        </form>
+      </div>
+      <div>
+        <p className="txt">Don’t have an account? Click here to <Link to="/parent-signup">sign up</Link></p>
+      </div>
+    </div>
+  );
+}
```
Comment (chuxmykel) [inline src/views/Login/index.js:34]: This is nice work @obayomi96. But in the future, I'll suggest we create reusable input elements so as to enable us reuse logic across the app.

Comment (obayomi96) [reply src/views/Login/index.js:34]: Thanks, I know that normally, but I also know very few people will use it, I had to put it down for now till our app gets more robust.

**Review by chuxmykel — APPROVED** [review]
Nice work buddy. LGTM

File: src/views/Login/index.js (hunk 1 above)
Comment (obayomi96) [reply src/views/Login/index.js:34]: I can implement it along with the new PR coming up for our landing page though.

Comment (chuxmykel) [reply src/views/Login/index.js:34]: All right man. Makes sense.

**Comment by obayomi96** [comment]
Thanks chux

**Review by EdustripeDev — APPROVED** [review]
LGTM

**Pull request merged**